libevm/interop/invoke_test.go

```diff
@@ -70,6 +70,15 @@ func TestInvoke(t *testing.T) {
 	if result.EvmError != "" {
 		t.Fatalf("vm error: %v", result.EvmError)
 	}
+	/* TODO: check test
```

Comment:
need to restore this test

Reply:
That test won't be possible in this way, because the output depends on the compiler version and some more things - more about that later.